### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by ChabVlad**

### CODE REVIEW

File: src/main/java/project/bookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Table(name = "names")
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/model/Book.java]:
```suggestion
@Table(name = "books")
```

File: src/main/java/project/bookstore/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+@Data
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/model/Book.java]:
```suggestion
@Getter
@Setter
```

File: src/main/java/project/bookstore/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(unique = true, nullable = false)
+    private String isbn;
+    @Column(nullable = false)
+    private Long price;
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/model/Book.java]:
```suggestion
    private BigDecimal price;
```

File: src/main/java/project/bookstore/repository/impl/BookRepositoryImpl.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import org.springframework.stereotype.Repository;
+import project.bookstore.model.Book;
+import project.bookstore.repository.BookRepository;
+
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+
+    @Override
+    public Book save(Book book) {
+        return null;
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/repository/impl/BookRepositoryImpl.java]: you should implement this method

File: src/main/java/project/bookstore/repository/impl/BookRepositoryImpl.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    }
+
+    @Override
+    public List findAll() {
+        return List.of();
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/repository/impl/BookRepositoryImpl.java]: same

File: src/main/java/project/bookstore/service/BookService.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package project.bookstore.service;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import project.bookstore.model.Book;
+
+@Service
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/service/BookService.java]:
```suggestion
```

File: src/main/java/project/bookstore/service/impl/BookServiceImpl.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
+package project.bookstore.service.impl;
+
+import java.util.List;
+import project.bookstore.model.Book;
+import project.bookstore.service.BookService;
+
+public class BookServiceImpl implements BookService {
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/service/impl/BookServiceImpl.java:13]:
```suggestion
@Service
public class BookServiceImpl implements BookService {
```

File: src/main/java/project/bookstore/service/impl/BookServiceImpl.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    @Override
+    public Book save(Book book) {
+        return null;
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/service/impl/BookServiceImpl.java]: implement this method

File: src/main/java/project/bookstore/service/impl/BookServiceImpl.java [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+    }
+
+    @Override
+    public List findAll() {
+        return List.of();
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/service/impl/BookServiceImpl.java]: implement this method

File: src/main/resources/application.properties [hunk 10]
```diff
@@ -0,0 +1,9 @@
+spring.application.name=BookStore
+spring.datasource.url=jdbc:h2:mem:testdb
```
Comment (Elena-Bruyako) [inline src/main/resources/application.properties]: h2 is used for test purpose, create mysql database

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comments

File: src/main/java/project/bookstore/BookStoreApplication.java [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BookStoreApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(BookStoreApplication.class, args);
+    }
+
```
Comment (okuzan) [inline src/main/java/project/bookstore/BookStoreApplication.java:12]:
```suggestion
    public static void main(String[] args) {
        SpringApplication.run(BookStoreApplication.class, args);
    }
```

File: pom.xml [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+                <configuration>
+                    <configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
+                    <consoleOutput>true</consoleOutput>
+                    <failsOnError>true</failsOnError>
+                    <linkXRef>false</linkXRef>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
```
Comment (okuzan) [inline pom.xml:140]:
```suggestion
```

**Review by okuzan — APPROVED** [review]

**Pull request closed without merging**